### ISSUE TITLE
AWS Default Region always set to ap-southeast-2

### ISSUE THREAD

**Issue by jordan-evans** [shinesolutions/aem-aws-stack-builder#187]
Hi,

On trying to stand this up in a different Region I can see the provisioning steps fail when the call is made to the below in the stack-init.sh script

```
echo "${label} Checking orchestration tags for ${component} component..."
/opt/shinesolutions/aws-tools/wait_for_ec2tags.py "$component"
```

On closer inspection I can see that .aws/credentials has the default region set to ap-southeast-2 (even though my config files have us-east-2 set). 

Not sure where this setup is happening as I can't see it being set in my base AMIs so not sure how to narrow it down anymore.

Everything works fine if I add the below to the top of the stack-init.sh script, but not a real solution

```
export AWS_DEFAULT_REGION=us-east-2
```

**Comment by cliffano** [comment]
@jordan-evans Thanks for reporting this issue. We'll have a look and update this thread with the finding.

**Comment by cliffano** [comment]
@jordan-evans We've spotted the one layer where the region defaulted to ap-southeast-2 and the user config was ignored. This has been fixed and it will be included in version 3.4.0 (scheduled to be released sometime this week).

Thanks again for bringing this to our attention. Glad to hear more use cases outside of Sydney region.
